docs/src/dedicated-gen-3/_index.md

```diff
@@ -1,79 +1,93 @@
-- Configuration changes on development environments (through your `services.yaml`, `routes.yaml`, and `.platform.app.yaml` files) are reflected on these {{% names/dedicated-gen-3 %}} clusters when you merge them.
-  It isn't necessary to open a support ticket to change production infrastructure like it is for {{% names/dedicated-gen-2 %}}.
-- You can deploy your application in any [supported cloud provider](../../development/regions.md#regions).
+  As a {{% names/dedicated-gen-3 %}} user,
+  you can take advantage of the [MariaDB Galera multi master cluster](https://mariadb.com/kb/en/galera-cluster/).
+  You can also resize your disk yourself to share your disk space across your apps and services as you need.
 
-## Differences from the Grid
+- **Seamless headless e-commerce with multi-app support.**
 
-Although {{% names/dedicated-gen-3 %}} adds plenty of features to your existing Grid applications, there are a few differences and limitations to be aware of when considering an upgrade. 
+- **Data synchronization from {{% names/dedicated-gen-3 %}} to Grid** through a single click from the Console.
 
-### Available services
+- **Better compliance.**<BR>
+  Your apps and configurations are fully independent from the machines they run on.
+  This allows Platform.sh to perform non-disruptive system upgrades to ensure you're always secure and compliant.
 
-Your application can connect to each service by referencing the exact same environment variables as a Grid environment.
-While the configuration of the service is performed by Platform.sh, the app configuration is the same and your code should be the same.
-See the [services documentation](../../add-services/_index.md) for service-specific details.
+## Storage
 
-The [services documentation](../../add-services/_index.md) states that not every service or version available on the Grid is available on {{% names/dedicated-gen-3 %}}.
-The table below shows the currently available services and their versions for {{% names/dedicated-gen-3 %}}.
+Each {{% names/dedicated-gen-3 %}} cluster comes with 50GB of storage per environment by default.
+This storage is intended for your data (databases, search indexes, user uploaded files, etc.)
+and you can subdivide it as you want.
 
-{{< gen-3-services >}}
+You can request more storage at any time.
 
-### Local mounts
+## Differences with the Grid
 
-Because you get a redundant infrastructure, note that local mounts are local to each Virtual Machine. Since you can't know which VM is going to handle a specific request, you also don't have a guarantee regarding which local mount is going to be used. Whether you actually want to use a local mount or in fact need to set up a [Network Storage mount](../add-services/network-storage.md) depends on your specific use-case.
+Although {{% names/dedicated-gen-3 %}} adds plenty of features to your existing Grid apps,
```

Comment:
I'm not sure if we should oppose DG3 and Grid. DG3 is mainly Grid on isolated VMs, with dedicated resources. We can explain that old version of services are not supported because of the high availability?